src/models/searchservice/response/directanswer/BaseFeaturedSnippetDirectAnswer.ts

```diff
@@ -0,0 +1,19 @@
+import { DirectAnswer } from './DirectAnswer';
+import { DirectAnswerType } from './DirectAnswerType';
+import { Snippet } from '../Snippet';
+import { BuiltInFieldType } from './BuiltInFieldType';
+import { EnumOrLiteral } from '../../../utils/EnumOrLiteral';
+
+/**
+ * A direct answer which was found within a document.
+ *
+ * @public
+ */
+export interface BaseFeaturedSnippetDirectAnswer<T = unknown> extends DirectAnswer<T> {
+  /** {@link DirectAnswerType}.FeaturedSnippet */
```

Comment:
should this be an `inheritDoc` instead of a `link`? as it is right now, this doc doesn't seem to be adding much value

Reply:
this was included before this pr. I don't think the dot notation here works anyway. inheritDoc sounds good. I can update